src/createServer.js

```diff
@@ -1,11 +1,24 @@
+/* eslint-disable no-console */
```

Comment:
Delete it.

Using `eslint-disable` is a bag practice. Try to avoid it here and in test tasks. Find a way to solve it without `eslint-disable`.(here I understand that it is redundant because you already deleted console.log).

But I need to worn you - how you should check you code in future test tasks.